app/main.py

```diff
@@ -1,34 +1,103 @@
-class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+# class Deck:
+#     def __init__(self, row, column, is_alive=True):
+#         pass
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
+    decks = []
```

Comment:
Same which I said about field in class Battleship, didn't mention before) I understand overall idea, that every ship should have different Decks, but that sounds not any flexible, as we gonna refactor a lot if we'll want to add any new logic inside Ship instances.